bootstrap.sh

```diff
@@ -1,23 +1,43 @@
-#! /bin/sh
+#! /bin/bash
```

Comment:
This path does not exist on my system. Do we really need bash features in this script?

Reply:
hm, I thought I'd seen a `shellcheck` warning about bash-specific features, but upon rechecking, I guess not; I'll revert it...

Reply:
ok, reverted in [deddd9a](https://github.com/autoconf-archive/autoconf-archive/pull/289/commits/deddd9ae7b75ccaffca08459262f7eb525141fea).

Reply:
OK, CI passed for [deddd9a](https://github.com/autoconf-archive/autoconf-archive/pull/289/commits/deddd9ae7b75ccaffca08459262f7eb525141fea), so I'm marking this conversation resolved.